src/browser/service-token.js

```diff
@@ -13,16 +13,19 @@ const deserializer = new Deserializer({ keyForAttribute: 'camelCase' })
  * @classdesc Helper class to request and manage a valid service token in a browser environment.
  *
  * @param {object} [options] the options
- *   - url {string}
+ *   - baseUrl {string}
+ *
+ * @throws {Error} if baseUrl is invalid
  */
 class ServiceTokenClient {
   constructor (options = {}) {
-    this.url = options.url || Config.serviceTokenUrl
+    this.baseUrl = options.baseUrl
```

Comment:
Because we use `baseUrl` in the node version. Should be non-breaking because the browser version still gets the url from `Config.serviceTokenUrl` (but in the BaseClient).